sentry_hipchat_ac/mentions.py

```diff
@@ -13,7 +12,12 @@
 RECENT_HOURS = 24 * 30
 
 
-cluster = make_rb_cluster(settings.SENTRY_REDIS_OPTIONS['hosts'])
+try:
+    from sentry.utils.redis import clusters
+    cluster = clusters.get('default')
+except ImportError:
```

Comment:
Wanna put a comment here on what version this is accommodating? Just like, `# Needed for Sentry < 8.2` or whatever so we know and can maybe one day hopefully remove.